lib/resources/example_json/soarian_examples_conformance.rb

```diff
@@ -0,0 +1,323 @@
+module Cerner
+  module Resources
+
+    SOARIAN_METADATA ||= {
+    "resourceType": "Conformance",
+    "publisher": "Cerner Corporation",
+    "date": "2017-03-15T11:54:23-04:00",
+    "copyright": "Copyright © Cerner Health Services, Inc. All rights reserved. This document contains confidential and/or proprietary information belonging to Cerner Health Services, Inc. and/or its related affiliates (Cerner) which may not be reproduced or transmitted in any form or by any means without the express written consent of Cerner. The Applications may contain embedded third party software including free and open source software (FOSS). Soarian's FOSS manual provides details and limitations that apply.",
+    "kind": "capability",
+    "software": {
+        "name": "Soarian Clinicals CCDS API",
+        "version": "01.00.0000"
+    },
+    "fhirVersion": "HAPI-1.3, FHIR DSTU2 - 1.0.2",
```

Comment:
@slvsridhar @dougpratt will need your input here.

In addition, in all resources, they're referencing the old FHIR sandbox, and they'll need to reference something that is either a good example people can try, or at least a non-prod endpoint that you can test with to generate accurate examples.